packages/@okta/vuepress-site/docs/concepts/oauth-openid/index.md

```diff
@@ -4,218 +4,231 @@ title: OAuth 2.0 and OpenID Connect Overview
 
-![flowchart/decision tree for choosing the correct OAuth 2.0 flow](/img/authorization/oauth_grant_flowchart.png)
+   <!--
+      Source image: https://www.figma.com/file/YH5Zhzp66kGCglrXQUag2E/%F0%9F%93%8A-Updated-Diagrams-for-Dev-Docs?type=design&node-id=4162%3A41793&mode=design&t=uYvvxmVdQRo6BVjR-1 oauth_grant_flowchart_new
+   -->
 
 </div>
 
 #### Is your client public?
 
-A client application is considered public when an end user could possibly view and modify the code. This includes Single-Page Apps (SPAs) or any mobile or native applications. In both cases, the application can't keep secrets from malicious users. Your client is considered confidential or private for server-side (web applications), which means your client can use client authentication such as a client secret.
+Single-Page Applications (SPAs), mobile, and platform-specific applications are **public** applications where end users can view and possibly modify their code. Any secrets in their code are open to malicious users. By comparison, server-side (web) applications are **confidential** or **private** applications. Private application clients can use client-side authentication methods such as client secrets.
 
-#### Is your client a SPA or native?
+#### Does your client use the redirect or embedded authentication model?
 
-If your client application is a SPA or a native application, you should use an authorization flow with PKCE, such as either the [Interaction Code flow with PKCE](#interaction-code-flow) or the [Authorization Code flow with PKCE](#authorization-code-flow-with-pkce). If you are doing a redirect flow to an Okta-hosted sign-in page, the Authorization Code flow with PKCE is recommended. If you want to embed the sign-in experience, the Interaction Code flow is recommended.
+If your client application is a SPA or a platform-specfic application that redirects its authentication requests to an Okta-hosted sign-in page ([the redirect model](/docs/concepts/redirect-vs-embedded/#redirect-authentication)), use the [Authorization Code flow with PKCE](#authorization-code-flow-with-pkce). If your application hosts the authentication flow itself ([the embedded model](/docs/concepts/redirect-vs-embedded/#embedded-authentication)), use the [Interaction Code flow](#interaction-code-flow).
 
 #### Does the client have an end user?
 
-If your client application is running on a server with no direct end user, then it can be trusted to handle credentials and use them responsibly. If your client application is only doing machine-to-machine interaction, then you should use the [Client Credentials flow](#client-credentials-flow).
+If your client application runs on a server with no direct end user, then it can be trusted to handle credentials and use them responsibly. If your client application is only doing machine-to-machine interaction, then you should use the [Client Credentials flow](#client-credentials-flow).
```

Comment:
```suggestion
If your client application runs on a server with no direct end user, then it can be trusted with its own credentials and use them responsibly. If your client application is only doing machine-to-machine interaction, then you should use the [Client Credentials flow](#client-credentials-flow).
```